sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -1773,21 +1774,12 @@ SUBROUTINE CALVOR(UWND,VWND,ABSV)
 !     
 !     LOOP TO COMPUTE ABSOLUTE VORTICITY FROM WINDS.
 !     
-      IF(MODELNAME  == 'RAPR') then
```

Comment:
@jaymes-kenyon The changes will have impact on results of rap and hrrr. There might be an upcoming hrrr implementation to include GTG4 updates. I suggest keeping the current settings for rap and hrrr. You may initialize ABSV as SPVAL when modelname is 'RAPR' and submodelname is 'MPAS'.